lib/matplotlib/transforms.py

```diff
@@ -2975,3 +2976,26 @@ def offset_copy(trans, fig=None, x=0.0, y=0.0, units='inches'):
         y /= 72.0
     # Default units are 'inches'
     return trans + ScaledTranslation(x, y, fig.dpi_scale_trans)
+
+
+class ScaledRotation(Affine2DBase):
+    """
+    A transformation that applies offset and direction
+    based on *trans_shift*.
+    """
+    def __init__(self, theta, trans_shift, is_radial):
+        super().__init__()
+        self._theta = theta
+        self._trans_shift = trans_shift
+        self._is_radial = is_radial
+        self._mtx = np.identity(3)
```

Comment:
This is (relatively) expensive, so it should copy from `IdentityTransform` like the other transforms do, but it might be simpler to set this to `None` as is done in `ScaledTranslation`